app/src/controller/front/Controller.php

```diff
@@ -2,15 +2,18 @@
 
 namespace App\Controller\Front;
 
+use App\Model\Repository\Repository;
 use Twig\Environment;
 
 abstract class Controller
 {
     protected Environment $twig;
+    protected $repository;
```

Comment:
not sure about $repository in the mother class. One controller could need to access different repositories ?
Twig seems mandatory, but maybe the repository part should be in the child class